packages/orchestrator-ui-components/src/components/WfoStartButton/WfoStartTaskComboBox.tsx

```diff
@@ -18,14 +18,16 @@ export const WfoStartTaskButtonComboBox = () => {
     const { data } = useGetTaskOptionsQuery();
     const taskOptions = data?.startOptions || [];
 
-    const comboBoxOptions: StartComboBoxOption[] = taskOptions.map(
-        (option) => ({
+    const comboBoxOptions: StartComboBoxOption[] = [...taskOptions]
+        .sort((taskA, taskB) =>
+            taskA.description.localeCompare(taskB.description),
+        )
+        .map((option) => ({
```

Comment:
Related to discussion below (@DutchBen ) -- maybe map first and then sort, just make sure the ones without description end up at the bottom (if they even exist)